packages/ckeditor5-dev-env/lib/release-tools/utils/getnewreleasetype.js

```diff
@@ -5,66 +5,98 @@
 
 'use strict';
 
-const { commitTypes, transform } = require( '../changelog/writer-options' );
+const conventionalCommitsParser = require( 'conventional-commits-parser' );
+const conventionalCommitsFilter = require( 'conventional-commits-filter' );
+const gitRawCommits = require( 'git-raw-commits' );
+const concat = require( 'concat-stream' );
+const parserOptions = require( './parser-options' );
+const { availableCommitTypes } = require( './transform-commit-utils' );
+const getPackageJson = require( './getpackagejson' );
+const versions = require( './versions' );
 
 /**
  * Returns a type (major, minor, patch) of the next release based on commits.
  *
+ * If given package has not changed, suggested version will be equal to 'skip'.
+ *
+ * @param {Function} transformCommit
+ * @param {Boolean} isDevPackage Is the function called in a repository
+ * with multiple packages (which is management by Lerna).
  * @returns {Promise}
  */
-module.exports = function getNewReleaseType() {
-	const conventionalRecommendedBump = require( 'conventional-recommended-bump' );
-	const parserOpts = require( '../changelog/parser-options' );
+module.exports = function getNewReleaseType( transformCommit, isDevPackage ) {
+	const packageJson = getPackageJson();
+	let fromVersion = versions.getLastFromChangelog();
+
+	if ( fromVersion ) {
+		if ( isDevPackage ) {
+			fromVersion = packageJson.name + '@' + fromVersion;
```

Comment:
I'm not sure about the fact that in one case we use `vx.y.z` notation and in the other case there's no `v`. I see that Lerna doesn't use `v` in its tags... so we can follow that, but then let's this logic extracted somewhere because I can see that the same code is repeated twice.